teoriasegnali/variabili-aleatorie.tex

```diff
@@ -543,9 +543,143 @@ \section{Variabile aleatoria gaussiana o normale}
 \end{equation}
 \end{proof}
 
+% La VA paretiana è stata introdotta nell'A.A. 16/17
```

Comment:
I will approve all down long 546-679

consider having equation counter with
\begin{equation} \end{equation}
instead of
\[ \]
to help numbering and referencing
like others equations